include/aspect/material_model/utilities.h

```diff
@@ -118,6 +118,23 @@ namespace aspect
             dRhodp (const double temperature,
                     const double pressure) const;
 
+            /**
+             * Returns a vector of all the column names in the lookup file
+             * that start with the character string vol_fraction_
+             */
+            std::vector<std::string>
+            phase_volume_column_names() const;
+
+            /**
+             * Returns the volume fraction of the phase_idth phase
+             * at a given temperature and pressure, where the phase_column_name
+             * corresponds to that given in phase_volume_column_names()[phase_id].
```

Comment:
Wait, but this doesn't really have anything to do with what phase_volume_column_names() does, phase_volume_column_names() could be changed, but this function would still return the volume fraction of the phase with the column index phase_id at a given temperature and pressure. I think the thing you're saying here really belongs into the documentation of std::vector<std::string> phase_column_names and std::vector<dealii::Table<2,double>> phase_volume_fractions (namely, that these belong together and are in the same order).

Reply:
Good point. I've removed this sentence and added documentation to phase_column_names and phase_volume_fractions.